src/test/java/com/microsoft/sqlserver/jdbc/unit/serial/DTOSerialTest.java

```diff
@@ -0,0 +1,176 @@
+package com.microsoft.sqlserver.jdbc.unit.serial;
+
+import java.sql.*;
+
+import org.junit.jupiter.api.Test;
+import org.junit.platform.runner.JUnitPlatform;
+import org.junit.runner.RunWith;
+
+import com.microsoft.sqlserver.jdbc.*;
+import com.microsoft.sqlserver.testframework.AbstractTest;
+
+import microsoft.sql.*;
+
+import java.io.*;
+
+import static org.junit.jupiter.api.Assertions.fail;
+
+
+@RunWith(JUnitPlatform.class)
+public class DTOSerialTest extends AbstractTest {
+    private static final String dateString = "2007-05-08 12:35:29.1234567 +12:15";

```

Comment:
Hard-coded value, so this will only ever test with this specific dateString? Might be a little hard to refactor considering the serialization test below but this probably isn't a good way to be testing these APIs.